makelib/helm.mk

```diff
@@ -84,6 +84,11 @@ $(HELM_OUTPUT_DIR)/$(1)-$(HELM_CHART_VERSION).tgz: $(HELM_HOME) $(HELM_OUTPUT_DI
 	fi
 	@$(OK) helm package $(1) $(HELM_CHART_VERSION)
 
+helm.repo.add:
+ifdef INTERNAL_MUSEUM_URL
+	@$(HELM) repo add $(INTERNAL_MUSEUM_NAME) $(INTERNAL_MUSEUM_URL) --username $(INTERNAL_MUSEUM_USER) --password $(INTERNAL_MUSEUM_PASS)
+endif
+
```

Comment:
@tnthornton how much effort would if be to make this more generic such that you can define a list of dependency repos (and optionally credential variables) then we add them all? Another option would be that if you just want to do this for one repo then you can just extend `build.init` target in its specific `Makefile`. That might be easier for now. See example in: https://github.com/upbound/up/blob/b812d4c29487e47ee73db6fe7778e7c19ec8843a/Makefile#L54

Reply:
>@tnthornton how much effort would if be to make this more generic such that you can define a list of dependency repos (and optionally credential variables) then we add them all?

Oh interesting. Let me play around with that and see.

>Another option would be that if you just want to do this for one repo then you can just extend build.init target in its specific Makefile. That might be easier for now. See example in: https://github.com/upbound/up/blob/b812d4c29487e47ee73db6fe7778e7c19ec8843a/Makefile#L54

I'll keep this in mind if I can't get the other suggestion to work.